teps/0105-remove-pipeline-v1alpha1-api.md

```diff
@@ -0,0 +1,320 @@
+---
+status: proposed
+title: Remove Pipeline v1alpha1 API
+creation-date: '2022-04-11'
+last-updated: '2022-04-11'
+authors:
+- '@abayer'
+---
+
+# TEP-0105: Remove Pipeline v1alpha1 API
+
+<!--
```

Comment:
would you mind removing the boilerplate and running the hack/update-toc.sh script please?

Reply:
The TOC is updated, I think?  See https://github.com/tektoncd/community/pull/676/files#diff-d146c6d6729ac071d523817bcfa4820d7d1e1ba4f9c867d17659d64e25ecd674R243

I left boilerplate in because it says `When your TEP is complete, all of these comment blocks should be removed` on the next line, and I wasn't sure if that meant I should leave all the boilerplate in until then or what. =)

Reply:
Oh sorry I was trying to suggest you delete the sections you're not using and then update the TOC, but if you'd like to keep the sections you're not using yet in place that's ok too. I think it would be good to remove the comment blocks for sections you've already filled in though; opened #681 to clarify

Reply:
👍 for that! =)